ui/component/or-dashboard-builder/src/settings/attribute-settings.ts

```diff
@@ -0,0 +1,620 @@
+              ${this.widgetConfig.textMappings.length
+                ? this.widgetConfig.textMappings.map(
+                    (m, idx) => html`
+                      <div class="mapping-item">
+                        <input
+                          type="text"
+                          placeholder="Original"
+                          .value=${m.from}
+                          @input=${(e: Event) => this.onTextFromChange(e, idx)}
+                        />
+                        <input
+                          type="text"
+                          placeholder="Ersetzen durch"
+                          .value=${m.to}
+                          @input=${(e: Event) => this.onTextToChange(e, idx)}
+                        />
+                        <or-mwc-input
+                          .type=${InputType.BUTTON}
+                          icon="delete-outline"
+                          @or-mwc-input-changed=${() => this.onRemoveTextMapping(idx)}
+                        ></or-mwc-input>
+                      </div>
+                    `
+                  )
+                : html`<div>Keine Text-Mappings definiert</div>`}
+            </div>
+          </div>
+
+          <div class="mapping-actions">
+            <or-mwc-input
+              .type=${InputType.BUTTON}
+              style="margin-top: 8px;"
+              label="Text-Mapping"
+              icon="plus"
+              @or-mwc-input-changed=${() => this.addTextMapping()}
+            ></or-mwc-input>
+          </div>
+        </settings-panel>
+
+        <settings-panel displayName="settings" expanded="true">
+          <div class="field">
+            <!-- Toggle helper text -->
+            <div class="switch-container">
+              <div class="switch-row">
+                <span><or-translate value="dashboard.showHelperText"></or-translate></span>
+                <or-mwc-input
+                  .type=${InputType.SWITCH}
+                  .value=${this.widgetConfig.showHelperText}
+                  @or-mwc-input-changed=${(ev: OrInputChangedEvent) => this.onHelperTextToggle(ev)}
+                ></or-mwc-input>
+              </div>
+
+              <div class="switch-row">
+                <span>Variable anzeigen</span>
+                <or-mwc-input
+                  .type=${InputType.SWITCH}
+                  .value=${this.widgetConfig.showVariable}
+                  @or-mwc-input-changed=${(ev: OrInputChangedEvent) => this.onShowVariableToggle(ev)}
+                ></or-mwc-input>
+              </div>
+
+              <div class="switch-row">
+                <span>Value anzeigen</span>
+                <or-mwc-input
+                  .type=${InputType.SWITCH}
+                  .value=${this.widgetConfig.showValue}
+                  @or-mwc-input-changed=${(ev: OrInputChangedEvent) => this.onShowValueToggle(ev)}
+                ></or-mwc-input>
+              </div>
+            </div>
+          </div>
+        </settings-panel>
```

Comment:
Same comment here on aligning text to the left, and controls to the right side of the window.